common/app/controllers/EmailSignupController.scala

```diff
@@ -190,7 +190,11 @@ class EmailSignupController(
       }
     }
 
-  def renderForm(emailType: String, listId: Int): Action[AnyContent] =
+  def renderFormWithParentComponent (emailType: String, listId: Int, parentComponent: String): Action[AnyContent] =
+    return renderForm(emailType,listId, Option(parentComponent))
```

Comment:
```suggestion
    renderForm(emailType,listId, Option(parentComponent))
```

I think the return is unecessary here maybe?